pkgs/top-level/all-packages.nix

```diff
@@ -22474,6 +22474,11 @@ in
 
   kops = callPackage ../applications/networking/cluster/kops { };
 
+  lguf-brightness = callPackage ../misc/lguf-brightness {
+    libusb = libusb1;
```

Comment:
Both of these can be dropped and just use `libusb1` and `ncurses5` as argument to the derivation directly.